templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -168,6 +176,18 @@ Parameters:
     ConstraintDescription: 'Must be a valid Bitbucket version number. For example: 5.0.0 or higher'
     Description: 'Version of Bitbucket to install. Please use version 5.0.0 or higher. Find valid versions at http://go.atlassian.com/bbs-releases'
     Type: String
+  BitbucketLicenseKey:
+    Default: ''
+    Description: (Optional) Provide a license key for Bitbucket Data Center if you have one. 
+    Type: String
+  BitbucketAdminPassword:
+    Description: (Optional) Password for the administrator ('admin') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol. Optional - If none provided, the template will use the value for the DBPassword.
+    NoEcho: True
+    Type: String
+  BitbucketDatasetURL:
+    Default: ''
+    Description: (Optional) Provide the HTTP/HTTPS URL for the dataset to restore.
```

Comment:
This might not be clear for customers, should we link https://confluence.atlassian.com/bitbucketserver/data-center-migration-957497834.html page?

Reply:
agreed , will link the page